### PR TITLE
Updates to mitigate FPs

### PR DESCRIPTION
**Pull request by jkamdjou**

### CODE REVIEW

File: detection-rules/attachment_pdf_file_with_embedded_content.yml [hunk 1]
```diff
@@ -36,7 +36,6 @@ source: |
         "rar_file",
         "tar_file",
         "xar_file",
-        "zip_file",
```
Comment (jkamdjou) [inline detection-rules/attachment_pdf_file_with_embedded_content.yml:39]: @ajpc500 any issues with removing this? seeing some FPs.

Comment (ajpc500) [reply detection-rules/attachment_pdf_file_with_embedded_content.yml:39]: go for it!

File: detection-rules/attachment_any_html_unsolicited.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ references:
   - "https://ired.team/offensive-security/defense-evasion/file-smuggling-with-html-and-javascript"
   - "https://sandbox.sublimesecurity.com?id=106315e9-166a-4e0f-946e-88ff6fd5f9fd"
 type: "rule"
-severity: "medium"
+severity: "low"
```
Comment (ianthiel) [inline detection-rules/attachment_any_html_unsolicited.yml:10]: should the "new sender" version of this rule also be `low`?

**Review by ianthiel — APPROVED** [review]
fwiw the removed rule generated a TP in a recent IMAP NUX test

File: detection-rules/attachment_any_html_unsolicited.yml (hunk 2 above)
Comment (jkamdjou) [reply detection-rules/attachment_any_html_unsolicited.yml:10]: from my exp "new sender" is more likely to be a TP than this one

**Pull request merged**